bin/import_mls_english.py

```diff
@@ -50,36 +49,11 @@ def main(root_dir):
                 )
                 audio_duration = read_ogg_opus_duration(audio_path)
                 # TODO: support other languages
-                transcript = (
-                    transcript.strip()
-                    .replace("-", " ")
-                    .replace("ñ", "n")
-                    .replace(".", "")
-                    .translate(
-                        {
-                            ord(ch): None
-                            for ch in (
-                                "а",
-                                "в",
-                                "е",
-                                "и",
-                                "к",
-                                "м",
-                                "н",
-                                "о",
-                                "п",
-                                "р",
-                                "т",
-                                "ы",
-                                "я",
-                            )
-                        }
-                    )
-                )
+                transcript = transcript.strip().replace("-", " ").replace(".", "")
                 subset_entries.append(
                     (
                         audio_path.relative_to(root_dir),
-                        audio_duration,
+                        str("{:.2f}".format(audio_duration)).replace(".", ""),
```

Comment:
Pandas performs serialization based on the types. What exactly is this line fixing?